plugins/module_utils/gpg_utils.py

```diff
@@ -232,6 +232,7 @@ def __get_recipients_from_pass_file(self, password_slug: str) -> List[str]:
         while base_path.as_posix() != "/":
             if os.path.isfile(base_path / self.pass_gpg_id_file):
                 break
+            print(f"no {self.pass_gpg_id_file} file found on {base_path}, traversing up")
```

Comment:
I'm also not sure whether this is the right way to log stuff in ansible modules, I don't think it is.

Reply:
since the module context is not available in the GPG utils, i'm not sure if there is a cleaner way without essentially injecting a logger